### DISCUSSION TITLE
Is there a way to manually add the 4.0.1 stable build to Blender Launcher

### DISCUSSION THREAD

**Discussion by eisenbruch** [Victor-IX/Blender-Launcher-V2#13] [General]
How can I download and add 4.0.1 to Blender Launcher manually?

When I download the portable files and drag that folder into my blender launcher\stable folder, it does not show up when I launch blender launcher

**Comment by shiena** [comment]
1. Copy unzipped blender folder to your blender launcher\custom folder
2. Select the Custom tab of blender-launcher
3. Select the User tab

<img width="640" alt="image" src="https://github.com/Victor-IX/Blender-Launcher-V2/assets/1295639/428707d1-4449-42da-8f19-f9c1f7d6829a">

**Reply by eisenbruch** [reply]
i put mine into the stable folder and after a few restarts it started showing up in blender launcher :)
